> Is the Linux CI failure something I should concern myself with?

The log says:

```
Run echo "Running --doctool to see if this changes the public API without updating the documentation."
Running --doctool to see if this changes the public API without updating the documentation.
If a diff is shown, it means that your code/doc changes are incomplete and you should update the class reference with --doctool.


diff --git a/doc/classes/@GlobalScope.xml b/doc/classes/@GlobalScope.xml
index 3dfb0f4..7c954df 100644
--- a/doc/classes/@GlobalScope.xml
+++ b/doc/classes/@GlobalScope.xml
@@ -213,10 +213,8 @@
 			</description>
 		</method>
 		<method name="error_string">
-			<return type="String">
-			</return>
-			<argument index="0" name="error" type="int">
-			</argument>
+			<return type="String" />
+			<argument index="0" name="error" type="int" />
 			<description>
 				Returns a human-readable name for the given error code.
 			</description>
Error: Process completed with exit code 1.
```

You need to [update the class reference](https://docs.godotengine.org/en/latest/community/contributing/updating_the_class_reference.html) to resolve this.

--- a/scene/gui/text_edit.cpp
+++ b/scene/gui/text_edit.cpp
@@ -2246,7 +2246,10 @@ void TextEdit::_gui_input(const Ref<InputEvent> &p_gui_input) {
 					selection.selecting_column = col;
 				}
 
-				if (!mb->is_double_click() && (OS::get_singleton()->get_ticks_msec() - last_dblclk) < 600 && cursor.line == prev_line) {
+				const int triple_click_timeout = 600;
+				const int triple_click_tolerance = 5;
+
+				if (!mb->is_double_click() && (OS::get_singleton()->get_ticks_msec() - last_dblclk) < triple_click_timeout && mb->get_position().distance_to(last_dblclk_pos) < triple_click_tolerance) {
 					// Triple-click select line.
 					selection.selecting_mode = SelectionMode::SELECTION_MODE_LINE;
 					_update_selection_mode_line();
@@ -2256,6 +2259,7 @@ void TextEdit::_gui_input(const Ref<InputEvent> &p_gui_input) {
 					selection.selecting_mode = SelectionMode::SELECTION_MODE_WORD;
 					_update_selection_mode_word();
 					last_dblclk = OS::get_singleton()->get_ticks_msec();
+					last_dblclk_pos = mb->get_position();
 				}
 
 				update();

--- a/scene/gui/text_edit.h
+++ b/scene/gui/text_edit.h
@@ -308,6 +308,7 @@ private:
 	String lookup_symbol_word;
 
 	uint64_t last_dblclk = 0;
+	Vector2 last_dblclk_pos;
 
 	Timer *idle_detect;
 	Timer *click_select_held;

